src/pages/nft/claim/index.vue

```diff
@@ -75,7 +75,10 @@
           target="_blank"
         >
           <div class="flex flex-col gap-[16px] mb-[24px]">
-            <NFTBookCoverWithFrame :src="NFTImageUrl" />
+            <NFTBookCoverWithFrame
+              :src="NFTImageUrl"
+              :wrapper-classes="['aspect-[1]']"
```

Comment:
This cannot override the default one, maybe a prop `aspectRatioClass` with default `aspect-[4/5]`

Reply:
Although I was able to successfully override the default aspect ratio, your suggestion is better
[💄 Add aspect ratio prop for CoverWithFrame](https://github.com/likecoin/liker-land/pull/1588/commits/72fed1c8f141e57a5f1456f8702c1887cb0f64b7)

Reply:
[🎨 Update prop name in BookCoverWithFrame](https://github.com/likecoin/liker-land/pull/1588/commits/05f9251d419bc9c33586c8972d7d48e45f4bbad6)